zebra/rib.h

```diff
@@ -148,6 +148,11 @@ struct route_entry {
 	uint32_t dplane_sequence;
 };
 
+#define RIB_SYSTEM_ROUTE(R)                                                    \
+	((R)->type == ZEBRA_ROUTE_KERNEL || (R)->type == ZEBRA_ROUTE_CONNECT)
+
```

Comment:
So we've got `RSYSTEM_ROUTE` in `rt.h` currently too. maybe all three should live in the same place? at least for me, it'd be clearer if the test on a zebra re was more like `RSYSTEM_ROUTE((R)->type)`

Reply:
What do you think about the updates? Or would you still prefer only passing the type directly?